src/Medidata.ZipkinTracer.Core/Models/Span.cs

```diff
@@ -1,16 +1,17 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 
 namespace Medidata.ZipkinTracer.Models
 {
     public class Span
     {
-        public long TraceId { get; set; }
+        public string TraceId { get; set; }
```

Comment:
@kenyamat @hkurniawan-mdsol @jcarres-mdsol  I discovered that the long type is not being used. Since TraceProvider already checked if this values are already correct and valid, these values may remain as strings and can be sent as is (string) to zipkin server.